images/win/Windows2022-Readme.md

```diff
@@ -4,35 +4,35 @@
 ***
 # Microsoft Windows Server 2022 Datacenter
 - OS Version: 10.0.20348 Build 707
-- Image Version: 20220515.1
+- Image Version: 20220522.1
 
 ## Enabled windows optional features
 - Windows Subsystem for Linux [WSLv1]
 
 ## Installed Software
 ### Language and Runtime
-- Bash 4.4.23(2)-release
+- Bash 4.4.23(1)-release
```

Comment:
Need to check why it was downgraded.

Reply:
Temporarily downgraded due to code-signing issue.